app/views/spree/shared/_social.html.erb

```diff
@@ -5,7 +5,7 @@
 
   <% Spree::AuthenticationMethod.available_for(spree_current_user).each do |method| %>
     <%= link_to(content_tag(:i, '', class: "icon-spree-#{method.provider.dasherize}-circled"),
-          spree.send("spree_user_#{method.provider}_omniauth_authorize_path"),
-          title: I18n.t('spree.sign_in_with', provider: method.provider)) if method.active %>
+          spree.send("spree_user_#{method.provider}_omniauth_authorize_path", r: rand),
+          title: Spree.t(:sign_in_with, provider: method.provider)) if method.active %>
```

Comment:
@VitaliyAdamkov I like this change, removing `I18n` from views is an improvement 👍 
But `Spree.t` has been deprecated in Solidus (see issue solidusio/solidus/issues/2773) and we're trying to remove it from the extensions as well. Can you please change this to something like `t('spree.sign_in_with' ...` ?

Reply:
done

Reply:
@VitaliyAdamkov thank you 👍. 
I have one last request, can you please squash the tow commits into a single one? I think this would make git history cleaner and tidier... thank you!

Reply:
done